popup.js

```diff
@@ -0,0 +1,251 @@
+var axios = require("axios");
+var ProgressBar = require("./progressbar.js");
+var Chart = require('chart.js');
+
+document.addEventListener('DOMContentLoaded', function() {
+
+    chrome.tabs.getSelected(null, function(tab) {
+        // Specify your actual API key here:
+        var APIKEY = API_KEY;
+        // Specify the URL you want PageSpeed results for here:
+        var URL_TO_GET_RESULTS_FOR = tab.url;
+        var API_URL = 'https://www.googleapis.com/pagespeedonline/v2/runPagespeed?';
+        var CHART_API_URL = 'http://chart.apis.google.com/chart?';
+
+
+        //event listener for the show suggestions button
+        var suggestionList = document.getElementsByTagName('ul');
+        var suggestionsButton = document.getElementById('suggestions');
+        var hideIt = false;
+        suggestionsButton.addEventListener('click', function() {
+            //toggling between hiding and showing suggestions
+            if (!hideIt) {
+                for (var i = 0; i < suggestionList.length; i++) {
```

Comment:
ok done! I didn't have to put them into one container, since the "suggestionsList" only had 1 item in it because we only insert 1 "ul"

I just removed the for loop entirely to get rid of that extra code.